worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/implementation/blocks/ChunkSectionedChunk.java

```diff
@@ -0,0 +1,45 @@
+package com.fastasyncworldedit.core.queue.implementation.blocks;
+
+import com.sk89q.worldedit.world.biome.BiomeType;
+
+public class ChunkSectionedChunk {
+
+    public Object[] sectionLocks;
+    protected int minSectionPosition;
+    protected int maxSectionPosition;
+    protected int sectionCount;
+
+    static BiomeType getBiomeType(
```

Comment:
I'm not a particular fan of having a random biome method here, I'm not really sure that this class is needed and I think it ends up adding more clutter than it aims to remove